cfe_internal/enterprise/federation/federation.cf

```diff
@@ -11,12 +15,12 @@ bundle agent federation_config
   classes:
     enterprise_edition.(policy_server|am_policy_hub)::
       "cfengine_internal_federation_enabled"
-        expression => fileexists("$(path)"),
+        expression => fileexists( $(path) ),
```

Comment:
@nickanderson is there a style guide for policy somewhere that says to have spaces here? Mostly just curious and want to read/bookmark said style guide. :)

Reply:
https://docs.cfengine.com/docs/3.12/guide-writing-and-serving-policy-policy-style.html

Looks like this style suggestion is missing from the guide. Avoid quoting variables if you can is advice I received from developers long ago because the parser can pick up on errors more easily.

Reply:
I still don't quite understand where I have to use quotes and not.

Reply:
secret knowledge